src/pages/info/EpisodeInfo.tsx

```diff
@@ -60,7 +62,7 @@ export default function EpisodeInfo({ item }: EpisodeInfoProps): JSX.Element {
               next={(): Character[] => characters}
               hasMore
               loader={<h4>{t('loading')}</h4>}
-              height={550}
+              height={height / 2.5}
```

Comment:
what do you mean '2.5'?

Reply:
here is 1/4 of page height

Reply:
Could we use flex and auto height?

Reply:
I guess if you want use 1/4 height of your page you can use flex